contracts/messagebuscontract/IMessageBus.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import "./Structs.sol";
+
+interface IMessageBus {
```

Comment:
pls comments to the interface as well.
Something like:
- this represents the smart-contract level API that allows dApps to send and receive messages from corresponding cross-chain contracts